apollo-router/src/plugins/telemetry/config_new/logging.rs

```diff
@@ -104,12 +105,36 @@ pub(crate) struct StdOut {
     pub(crate) enabled: bool,
     /// The format to log to stdout.
     pub(crate) format: Format,
+    /// Log rate limiting. The limit is set per type of log message
+    pub(crate) rate_limit: Option<RateLimit>,
 }
+
 impl Default for StdOut {
     fn default() -> Self {
         StdOut {
             enabled: true,
             format: Format::default(),
+            rate_limit: None,
+        }
+    }
+}
+
+#[derive(Deserialize, JsonSchema, Clone, Debug)]
+#[serde(deny_unknown_fields, default)]
+pub(crate) struct RateLimit {
+    /// Number of log lines allowed in interval per message
+    pub(crate) count: u32,
```

Comment:
`count` is sort of non-descript.
Maybe `max_events` ? Still not great, but....

Reply:
in the traffic shaping's rate limiting we use "capacity", I'll use that to keep consistency

Reply:
fixed in [5dc5aa2](https://github.com/apollographql/router/pull/4517/commits/5dc5aa25977a25d9164b9b2bc4628ce23bc924d5)